13-transport.Rmd

```diff
@@ -513,7 +513,7 @@ Routing can be time and memory-consuming, resulting in large objects, due to the
 We will therefore filter the desire lines before calculating routes in this section.
 
 Cycling is most beneficial when it replaces car trips.
-Short (around 5 km, which can be cycled in 15 minutes at a speed of 20 km/hr) have a relatively high probability of being cycled, and the maximum distance increases when trips are made by [electric bike](https://www.sciencedirect.com/science/article/pii/S0967070X21003401) [@lovelace_propensity_2017].
+Short (around 5 km, which can be cycled in 15 minutes at a speed of 20 km/hr) have a relatively high probability of being cycled, and the maximum distance increases when trips are made by [electric bike](https://doi.org/10.1016/j.tranpol.2021.11.019) [@lovelace_propensity_2017].
```

Comment:
Marginal whether we need  that  URL and arguably should be reference instead if we do keep it for consistency. My bad!

Reply:
This url was found broken by the automatic system. But as you said, maybe it is not even needed and the reference could be better here.

Reply:
I suggest remove the link. There are loads of studies on E-bikes.